Source/Scene/Cesium3DTile.js

```diff
@@ -136,7 +150,10 @@ define([
             var type = url.substring(url.lastIndexOf('.') + 1);
             var contentFactory = Cesium3DTileContentProviderFactory[type];
 
-            if (defined(contentFactory)) {
+            if (type === 'json') {
```

Comment:
We removed `content.type`: https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/5

We can add it back, but it is redundant, except for the cases where the file wouldn't have an extension.  Can you point me to something that showing this is the right practice in this context - as opposed to a generic web request?  It feels like overkill.